pkg/reconciler/apis/apibinding/apibinding_controller.go

```diff
@@ -270,15 +271,28 @@ func (c *controller) enqueueAPIResourceSchema(obj interface{}) {
 		return
 	}
 
-	klog.V(2).Infof("Mapping APIResourceSchema %q", key)
 	apiExports, err := c.apiExportsIndexer.ByIndex(indexAPIExportsByAPIResourceSchema, key)
 	if err != nil {
 		runtime.HandleError(err)
 		return
 	}
 
-	for _, apiExport := range apiExports {
-		c.enqueueAPIExport(apiExport)
+	for _, obj := range apiExports {
+		apiExport := obj.(*apisv1alpha1.APIExport)
```

Comment:
It looks like the reimplementation of the content inside the loop, here and in enqueueAPIExport, is solely to be able to log why an APIBinding was enqueued? If so, what if we changed `enqueueAPIBinding` to `enqueueAPIBinding(obj interface{}, why string)` and have the reason appended? Could do similar for the other enqueue functions as needed.

Not terribly critical by any means, but I'd like to avoid the reimplementations if possible.

Reply:
some log string prefix arg would help

Reply:
done